DFC.ServiceTaxonomy.Cypher/Configuration/EndpointConfiguration.cs

```diff
@@ -0,0 +1,11 @@
+namespace DFC.ServiceTaxonomy.Cypher.Configuration
+{
+    public class EndpointConfiguration
```

Comment:
Is there a reason for not using the existing `EndpointConfiguration` and `Neo4jConfiguration` types?